### PR TITLE
Add ingress-rule plugin to krew index

### PR DESCRIPTION
**Pull request by pragaonj**
First release of `ingress-rule` plugin.

`ingress-rule` allows the configuration of ingress rules via command line.

Plugin installation has been tested locally.

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

**Comment by pragaonj** [comment]
I signed it

File: plugins/ingress-rule.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          arch: amd64
+      uri: https://github.com/pragaonj/ingress-rule-updater/releases/download/v0.1.0/ingress-rule_v0.1.0_windows_amd64.zip
+      sha256: 4159631f4b0fadf6b8d8b9092b1533de9e69ab13abf2f32c6cb473adbebb7dcb
+      files:
+        - from: "/ingress-rule.exe"
+          to: "."
+        - from: LICENSE
+          to: "."
+      bin: "ingress-rule.exe"
+  shortDescription: Update ingress rules via command line.
```
Comment (ahmetb) [inline plugins/ingress-rule.yaml:44]: (you can address this in the next version) 

- ingress → Ingress
- no trailing period `.`

File: plugins/ingress-rule.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  homepage: https://github.com/pragaonj/ingress-rule-updater
+  description: |
+    Add/remove rules to/from a kubernetes ingress via command line.
```
Comment (ahmetb) [inline plugins/ingress-rule.yaml:47]: (you can address this in the next version) 

- kubernetes → Kubernetes 
- ingress → Ingress

mostly cosmetics however.

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks!

Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Pull request merged**